libraries/radwordsprocessing/concepts/fields/sequence-field.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Sequence Field
+page_title: Sequence Field
+description: MergeField field is a Field element that contains a reference to a data field by its name.
+slug: radwordsprocessing-concepts-sequence-field
+tags: sequence,field
+published: True
+position: 8
+---
+
+# Sequence Field
+
+
+
+SequenceField is a [Field]({%slug radwordsprocessing-concepts-fields%}) element that sequally numbers chapters, tables, figures, and other items in the document. 
```

Comment:
It is a SeqField, not SequenceField. It is written both ways in MS documentation, but we (the team) decided to name it SeqField in our API.